versions.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
```

Comment:
Looks like this file snuck in. I'll make a note to update our `.gitignore` file.